drv/ArrayList.drv

```diff
@@ -571,6 +571,20 @@ public class ARRAY_LIST KEY_GENERIC extends ABSTRACT_LIST KEY_GENERIC implements
 		return modified;
 	}
 
+	@Override
+	public boolean retainAll(final COLLECTION c) {
+		final KEY_TYPE[] a = this.a;
```

Comment:
for consistency with the `removeIf` etc., I guess `java.util.Objects.requireNonNull(c);`

Reply:
Should I also add this to removeAll? because this was just a removeAll copy without the "!" at the contains check.

Reply:
Yeah I think so.